pkg/client-sdk/covenantless_client.go

```diff
@@ -1433,11 +1299,81 @@ func (a *covenantlessArkClient) completeUnilateralExit(
 	return ptx.B64Encode()
 }
 
+func (a *covenantlessArkClient) Select(
```

Comment:
Doesn't need to be exported.
```suggestion
func (a *covenantlessArkClient) selectFunds(
```